### ISSUE TITLE
atom-text-editor.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by guilhermemarconi** [sn4il/indent-tooltip#3]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
atom-text-editor.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/text-editor-element.js:68:18)
Object.updateTooltip (/Users/marconi/.atom/packages/indent-tooltip/lib/indent-tooltip.coffee:106:18)
Object.enable (/Users/marconi/.atom/packages/indent-tooltip/lib/indent-tooltip.coffee:34:12)
Object.toggle (/Users/marconi/.atom/packages/indent-tooltip/lib/indent-tooltip.coffee:26:21)
atom-workspace.subscriptions.add.atom.commands.add.indent-tooltip:toggle (/Users/marconi/.atom/packages/indent-tooltip/lib/indent-tooltip.coffee:19:26)
CommandRegistry.handleCommandEvent (<embedded>:18796:35)
```

**Comment by ilkome** [comment]
+1

**Comment by sn4il** [comment]
Thank you for reporting the bug. I've just fixed it in v0.2.2.

P.S.: Sorry for late fix.
